packages/generate-pdf/src/index.ts

```diff
@@ -135,18 +136,18 @@ async function getPdfMakeContentFromObject(
   extraTranslations: ExtraTranslations = null,
 ): Promise<TDocumentDefinitions['content']> {
   const refSchemaUrl = new URL(certificate.RefSchemaUrl);
-  const schemaConfig = getSchemaConfig(refSchemaUrl);
+  const schemaConfig: SchemaConfig = getSchemaConfig(refSchemaUrl);
```

Comment:
Is that type assertion really needed ? getSchemaConfig should return SchemaConfig already

Reply:
For some reason the type is reported as `any` when I hover over `schemaConfig`, and no return type is reported by `getSchemaConfig` in `generate-pdf`. The types are all correctly set in `getSchemaConfig`'s declaration in `utils`, so I assume it's just a problem with Lerna. Would you prefer that I delete the type assertion?

Reply:
smell like a build / dependency link issue indeed. Maybe rebuilding and bootstrapping could fix that undesired `any`